### PR TITLE
DiffEdit: Editing Images using Generative AI

### PR DESCRIPTION
**Pull request by JonnySpruce**
This is a blog post put together by myself and  @jstrong-scottlogic, as part of the bench project on _Practical Deep Learning for Coders_.

Please add a direct link to your post here:

https://jonnyspruce.github.io/blog/2023/12/13/diffedit.html

Have you (please tick each box to show completion):

 - [x] Added your blog post to a single category?
 - [x] Added a brief summary for your post? Summaries should be roughly two sentences in length and give potential readers a good idea of the contents of your post.
 - [x] Checked that the build passes?
 - [x] Checked your spelling (you can use `npm spellcheck` if that's your thing)
 - [x] Ensured that your author profile contains a profile image, and a brief description of yourself? (make it more interesting than just your job title!)
 - [x] Optimised any images in your post? They should be less than 100KBytes as a general guide. 
 > *Note:* a couple of images are still above 100KB (>200KB however) - let me know if this needs to be reduced further.

Category:
 - Artificial Intelligence - Chris Price has reviewed this blog post multiple times during writing, and suggested that I ask @ColinEberhardt for final review.

### CODE REVIEW

File: _posts/2023-12-13-diffedit.md [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+![png]({{ site.github.url }}/jspruce/assets/diffedit_12_0.jpg "The results of the DiffEdit process")
+    
+
+
+While many diffusion models can be used in an image to image (img2img) pipeline, it is fairly limited compared to the DiffEdit technique. In img2img, all parts of the image will be changed by this process, you cannot limit the modifications of the image to a specific area. In our image above you can see that the background is unchanged between our original image and the generated zebra image.
+
+The DiffEdit paper utilises the noise prediction which a diffusion model provides to us to automatically limit edits to certain parts of an image. It does this in three main steps which are summarised in the pipeline chart below:
+
+
+![svg]({{ site.github.url }}/jspruce/assets/diffedit_steps.svg "DiffEdit pipeline steps")
```
Comment (ColinEberhardt) [inline _posts/2023-12-13-diffedit.md:33]: This is a broken link, currently the file sits in the `_posts` folder:

image

**Review by ColinEberhardt — CHANGES_REQUESTED** [review]
Great post, thanks - one small issue to resolve.

File: _posts/2023-12-13-diffedit.md (hunk 1 above)
Comment (JonnySpruce) [reply _posts/2023-12-13-diffedit.md:33]: Thanks for spotting that - should be resolved now!

**Comment by ColinEberhardt** [comment]
Thanks @JonnySpruce @jstrong-scottlogic  - great post, just merged

**Pull request merged**